### PR TITLE
convert page size string values to ints

### PR DESCRIPTION
**Pull request by camallen**
allow page size selection params to cascade into the subject selector.

### CODE REVIEW

File: lib/subject_selector.rb [hunk 1]
```diff
@@ -58,7 +58,10 @@ def missing_subject_set_error
   end
 
   def default_page_size
```
Comment (marten) [inline lib/subject_selector.rb]: I think "default_page_size" is a bit of a misnomer given that it can be overridden using params.

**Comment by marten** [comment]
Minor naming niggle, but LGTM

**Comment by camallen** [comment]
this isn't right...fix coming

**Comment by marten** [comment]
bnfge-iccaaqpyi jpg-large

**Comment by camallen** [comment]
just in case i didn't get the message the first time.

**Pull request merged**